ansible/roles/master/defaults/main.yml

```diff
@@ -1,7 +1,15 @@
-kube_master_insecure_port: 8080
```

Comment:
From my understanding (understand running ``grep -rn "kube_master_insecure_port"``) the ``kube_master_insecure_port`` is used only when the kubelet is run on master node. Otherwise ``kube_master_api_port`` is used outside of master.